docs/flank_mixpanel_metrics.md

```diff
@@ -0,0 +1,57 @@
+# Mixpanel events in Flank
+
+Flank currently tracking the following events:
+
+- configuration
+- bundle id and package name with information about the device type
+- total cost
+- cost per device type
+- test duration
+- flank version
+
+Every event contains a session id and project id. By ```project id``` you can find all events from every execution with specific ```project id```. By ```session id``` you can find events from specific
```

Comment:
The end of the sentence appears to be not finished.
EG:
specific...?

Reply:
Fixed, I removed the redundant new line.

Reply:
👍